pkg/util/globalerror/errors.go

```diff
@@ -60,3 +64,18 @@ func (id ID) Message(msg string) string {
 func (id ID) MessageWithLimitConfig(flag, msg string) string {
 	return fmt.Sprintf("%s (%s%s). You can adjust the related per-tenant limit by configuring -%s, or by contacting your service administrator.", msg, errPrefix, id, flag)
 }
+
+func (id ID) MessageWithLimitConfigs(msg, flag string, addFlags ...string) string {
```

Comment:
Please add few unit tests on this function, to test the cases:
- 1 flag
- 2 flags
- 3 flags